src/Forms/Main.cs

```diff
@@ -7402,18 +7408,43 @@ private void ContextMenuStripListviewOpening(object sender, System.ComponentMode
                     toolStripMenuItemWebVTT.DropDownItems.Add(_language.Menu.ContextMenu.WebVTTRemoveVoices, null, WebVTTRemoveVoices);
                 }
             }
-            else if ((format.Name == "Nuendo" && SubtitleListview1.SelectedItems.Count > 0))
+            else if (formatType == typeof(CsvNuendo) && SubtitleListview1.SelectedItems.Count > 0)
             {
-                toolStripMenuItemWebVTT.Visible = false;
-                var styles = GetNuendoStyles();
-                setStylesForSelectedLinesToolStripMenuItem.DropDownItems.Clear();
-                foreach (var style in styles)
+                // characters
+                actors = GetNuendoStyles();
+                setActorForSelectedLinesToolStripMenuItem.DropDownItems.Clear();
+                foreach (var character in actors)
                 {
-                    setStylesForSelectedLinesToolStripMenuItem.DropDownItems.Add(style, null, NuendoSetStyle);
+                    if (string.IsNullOrEmpty(character))
+                    {
+                        continue;
+                    }
+                    if (actors.Contains(character))
+                    {
+                        continue;
+                    }
+                    setActorForSelectedLinesToolStripMenuItem.DropDownItems.Add(character, null, NuendoSetStyle);
                 }
-                setStylesForSelectedLinesToolStripMenuItem.Visible = styles.Count > 1;
+                actors.Sort();
+
+                if (actors.Count > 0)
+                {
+                    var tss = new ToolStripSeparator();
+                    UiUtil.FixFonts(tss);
+                    setActorForSelectedLinesToolStripMenuItem.DropDownItems.Add(tss);
+                }
+                setActorForSelectedLinesToolStripMenuItem.DropDownItems.Add(_language.Menu.ContextMenu.NewActor, null, SetNewActor);
+                if (actors.Count > 0)
+                {
+                    setActorForSelectedLinesToolStripMenuItem.DropDownItems.Add(_language.Menu.ContextMenu.RemoveActors, null, RemoveActors);
+                }
+
                 toolStripMenuItemAssStyles.Visible = false;
-                setStylesForSelectedLinesToolStripMenuItem.Text = _language.Menu.ContextMenu.NuendoSetStyle;

```

Comment:
for Nuendo subtitle format, we want to update the `Actor` (**Character**) instead of style property